src/Views/FirmwareView.vala

```diff
@@ -47,7 +47,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
 
         placeholder_alert_view = new Granite.Widgets.AlertView (
             _("Checking for Updates"),
-            _("Connecting to the firmware service and searching for updates."),
+            _("Connecting to the service to look for new firmware…"),
```

Comment:
This sounds less natural to me. Your device already has some firmware, so it's not inaccurate to say that we're looking for updates

Reply:
I'll simplify it to what actually happens.
Originally I thought to just add the ellipsis, but firmware upgrades are not updates.

Reply:
I'm curious what you feel is the difference between "upgrade" and "update"?

Reply:
Update to pull in the list of packages, and upgrade to install those.
I made an effort to avoid it all though.